modules/tinyexr/image_loader_tinyexr.cpp

```diff
@@ -175,9 +190,11 @@ Error ImageLoaderTinyEXR::load_image(Ref<Image> p_image, FileAccess *f, bool p_f
 		exr_tiles = exr_image.tiles;
 	}
 
+	//print_line("reading format: " + Image::get_format_name(format));
```

Comment:
Leftover `print_line`, should probably be removed.

Reply:
I don't entirely trust that its working, so I left it there

Reply:
(though I tested it a lot)